tests/t332-basis.c

```diff
@@ -1,37 +1,38 @@
 /// @file
-/// Test GetDiv and BasisApply for a 2D Quad non-tensor H(div) basis
-/// \test Test GetDiv and BasisApply for a 2D Quad non-tensor H(div) basis
+/// Test div with a 2D Quad non-tensor H(div) basis
+/// \test Test div with a 2D Quad non-tensor H(div) basis
 #include <ceed.h>
 #include <math.h>
 
 #include "t330-basis.h"
 
 int main(int argc, char **argv) {
   Ceed              ceed;
-  const CeedInt     num_nodes = 4, Q = 3, dim = 2, num_qpts = Q * Q;
-  CeedInt           num_comp = 1;                // one vector componenet
-  CeedInt           P        = dim * num_nodes;  // dof per element!
+  const CeedInt     Q = 3, dim = 2, num_qpts = Q * Q, elem_nodes = 4;
+  const CeedInt     P = dim * elem_nodes;
   CeedBasis         b;
   CeedScalar        q_ref[dim * num_qpts], q_weights[num_qpts];
-  CeedScalar        div[P * num_qpts], interp[P * dim * num_qpts];
+  CeedScalar        interp[dim * P * num_qpts], div[P * num_qpts];
   CeedVector        X, Y;
-  const CeedScalar *y, *x;
+  const CeedScalar *y, *x, *div2;
```

Comment:
```suggestion
  const CeedScalar *y, *x, *div_in_basis;
```
maybe? Open to other names, the 2 suffix is just not as clear

Reply:
[2a6c126](https://github.com/CEED/libCEED/pull/1156/commits/2a6c12603dc224f416582469a85baf5764d87787)